api/src/templates/new_member.body.html

```diff
@@ -1,167 +1,194 @@
+
+{% block language clarification %}
+<div class="text-box">
+    <h3 > About language</h3>
+    <p>The primary language on Stockholm Makerspace is Swedish, but you are very welcome to use English as well. Newsletters and important announcements as well as our courses are as a rule available in English. If you need help to translate something from Swedish to English, don’t hesitate to ask other members or post about it on Slack.</p>
+</div>
+<hr>
+{% enbblock %}
 
-<h3>TVÅ TYPER AV MEDLEMSKAP</h3>
-<p>
-    <b>Grundmedlem</b><br/>
-    Som medlem i föreningen så är du välkommen på kurser och workhops som vi arrangerar, samt årsmötet. Vill du utföra självständigt arbete i lokalen så ska du skaffa makerspace access.
-</p>
+{% block swedish %}
+<h2>SVENSKA</h2>
+<p>Hejsan! Välkommen till Stockholm Makerspace! &#128515;<br> Vi hoppas du uppskattade introkursen och är redo att påbörja din kreativa resa med oss.</p>
 
-<p>
-    <b>Makerspace Access</b><br/>
-    För medlemmar som har Makerspace Access så är lokalen öppen dygnet runt, året runt, men för att komma in första gången så måste du besöka en medlemsintroduktion. Medlemsintroduktioner äger rum ungefär varannan vecka, och datum hittar du i vårt <a href="https://makerspace.se/kalendarium">kalendarium</a>. För få access måste du skriva under ett avtal där du ålägger dig att följa våra <a href="https://wiki.makerspace.se/Lokalregler">lokalregler</a>, samt visa ID.
-</p>
+<p>Som ny medlem har du förmodligen en del frågor. I detta mail kommer du få en sammanfattning av vad du fick höra på introkursen och länkar till några onlineresurser som hjälper dig att sätta igång.</p>
 
-<h3>Jag vill gå kurs! När är det kurs?</h3>
-<p>All verksamhet bedrivs av volontärer, som du och jag. Vi hjälps åt att lära varandra maskinerna så att alla kan utföra självständigt arbete. Man delar vad man kan, och ber om hjälp om man kör fast. Några saker måste man gå kurs för att kunna använda, till exempel: svetsrummet, laserskäraren, brodyrmaskinen och resin-printern. Kurser annonseras i <a href="https://www.facebook.com/groups/makerspace.se">Facebookgruppen</a> och på <a href="https://wiki.makerspace.se/Slack">Slack</a> och läggs in i kalendern. Det finns ingen kö, först till kvarn gäller.
+<h3>Dina medlemsuppgifter:</h3>
+<p>Namn: {{member.firstname}} {{member.lastname}}<br/>
+    Epost: {{member.email}}<br/>
+    Medlemsnummer: {{member.member_number}}<br/>
 </p>
 
-<h3>Jag är jätteduktig på en sak. Får jag hålla kurs i saken?</h3>
-<p>Absolut! Emaila styrelsen på <a href="mailto:info@makerspace.se">info@makerspace.se</a> så pratar vi upplägg, anmälan och sätter in kursen i kalendariet. Om du är osäker på om det finns en efterfrågan inom just ditt kunskapsområde kan du kolla av intresse genom att posta om saken i Facebookgruppen eller på Slack.</p>
+<p>Vi kommer använda denna epostadress för att nå ut till dig. Om detta inte stämmer, vänligen kontakta <a href="mailto:info@makerspace.se">info@makerspace.se</a>.</p>
```

Comment:
Om man fick mailet så borde ju saker fungera för det mesta. Tror att större delen av gångerna det inte fungerar så får medlemmen inte mailet alls. Så tycker inte att det här stycket behövs. Tycker inte det är värt att göra mailet längre för det.

Reply:
The reasoning for this line is the same as for the previous conversation. Still have no strong feelings about this. If you feel it is unnecessary, we remove it.